css/main.css

```diff
@@ -20,7 +20,12 @@ palette from generator 2
     url("/static/fonts/ttf/Overpass-LightItalic.ttf");
 }
 
+a > img {
+  margin-left: 1rem;
+}
+
 body {
+  font-family: "Roboto Mono", sans-serif;
```

Comment:
suggestion: Inconsolata

Reply:
looks a bit thin. but up to you
![bilde](https://user-images.githubusercontent.com/35314375/120052613-66bec280-c026-11eb-982d-c3199ac83ad4.png)